src/main/kotlin/uk/gov/dluhc/printapi/rest/TemporaryCertificateController.kt

```diff
@@ -84,6 +95,20 @@ class TemporaryCertificateController(
             userId
         )
         return temporaryCertificateService.generateTemporaryCertificate(eroId, dto).also {
+            if (it.contents.size > MAX_RESPONSE_FILE_SIZE && generateTemporaryCertificateRequest.allowLargeResponse != true) {
+                val s3Path = "temporary_certificates/${dto.gssCode}/${dto.applicationReference}.pdf"
```

Comment:
I think the bucket has versioning enabled, and they probably shouldn't be generating multiple certificates for one application anyway, but could we just add a unique ID into this path anyway?